data/card_catalogue/upgradeable-1.json

```diff
@@ -684,14 +422,14 @@
     "upgrade": "HoloKicker",
     "tier": "green",
     "details": {
-      "rules": "Summon an immobile monk until end of turn who punches characters who come within range for 6 damage.",
+      "rules": "Summon an immobile monk until end of turn who punches characters who come within range",
```

Comment:
This set is weird AF. I'm thinking to maybe cut it and revisit summons later. Thoughts @elliottomlinson ?

Reply:
Sounds good to me, always thought it was a strange set. 

For better or for worse I'll remember Holomonk for being the first bit of art to use the "signature look". Would be down to get him back in when we revisit summons.

![holomonk](https://github.com/elliottomlinson/rpcg/blob/master/assets/core/card/illustration/Holomonk.png?raw=true)

Reply:
It's a great piece of art for sure. We'll find a use for it eventually